integration/test_fulltext.py

```
@@ -1952,6 +1952,62 @@ def test_return_clause(self):
+        for doc_id in range(1, 4):
+            args = []
+            for k, v in field_values.items():
+                args.extend([k, f"{v}_doc{doc_id}"])
+            client.execute_command("HSET", f"doc:{doc_id}", *args)
+        IndexingTestHelper.wait_for_backfill_complete_on_node(client, "idx_content")
+
+        # Path 1: FetchSpecificFields - RETURN 2 fields (2 <= 10/2=5)
+        result = client.execute_command("FT.SEARCH", "idx_content", "value1_doc1",
+                                        "RETURN", "2", "f1", "f2")
+        assert result[0] == 1
+        doc_fields = dict(zip(result[2][::2], result[2][1::2]))
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)